Gems/Atom/Feature/Common/Assets/Config/Platform/Linux/AtomShaderCapabilities.json

```diff
@@ -0,0 +1,17 @@
+{
+    "Type": "JsonSerialization",
+    "Version": 0,
+    "ClassName": "CapabilitiesConfigFile",
+    "ClassData":
+    {
+        "DescriptorCounts":
+        {
+            "Sets": 80,
+            "Spaces": 8,
+            "Samplers": 8,
+            "Textures": -1,
+            "Buffers": -1
+        },
+        "MaxSpaces": 4
```

Comment:
Dont add this file. This number enforcers SRG merging due to restrictions on the number oof SRGs allowed on Android.

Reply:
Will Remove